src/Uno.UI/UI/Xaml/DependencyObjectStore.cs

```diff
@@ -1647,7 +1647,8 @@ DependencyPropertyValuePrecedences newPrecedence
 
 				if (frameworkPropertyMetadata.Options.HasAffectsRender())
 				{
-					actualInstanceAlias.InvalidateRender();
+					// Uno TODO: What should we do here?
+					//actualInstanceAlias.InvalidateRender();
```

Comment:
For now I would redirect this to the `Compositor.InvalidateRender()` (even if I'm about to remove that method in my branch).

Reply:
@dr1rrb Where is this method? I'm not able to find it

Reply:
[`ElementCompositionPreview.GetElementVisual(actualInstanceAlias)`](https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/Hosting/ElementCompositionPreview.cs#L16)[`.Compositor`](https://github.com/unoplatform/uno/blob/master/src/Uno.UWP/UI/Composition/CompositionObject.cs#L25)[`.InvalidateRender()`](https://github.com/unoplatform/uno/blob/master/src/Uno.UWP/UI/Composition/Compositor.cs#L106)

```suggestion
					if (actualInstanceAlias is UIElement elt)
					{
						Windows.UI.Xaml.Hosting.ElementCompositionPreview.GetElementVisual(elt).Compositor.InvalidateRender();
					}
```